AI/MIGraphX/Quantization/Readme.md

```diff
@@ -0,0 +1,50 @@
+
+# MIGraphX - Torch Examples
+
+# Summary
+The examples in this subdirectory showcase the functionality of the [MIGraphX](https://github.com/ROCmSoftwarePlatform/torch_migraphx/tree/master) library. The examples build on Linux for the ROCm (AMD GPU) backend.
```

Comment:
```suggestion
The examples in this subdirectory showcase the functionality of the [MIGraphX](https://github.com/ROCmSoftwarePlatform/torch_migraphx/tree/master) library. The examples build on Linux for the ROCm (AMD GPU) backend.
```
 Last sentence is too technical, let's rewrite.  AMD GPU backend term does not need to be used.

Reply:
Fixed. Please review the updated Summary section in the Readme file.